src/commands/api/getAzureContainerAppsApiProvider.ts

```diff
@@ -9,7 +9,7 @@ import type * as api from "./vscode-azurecontainerapps.api";
 
 export function getAzureContainerAppsApiProvider(): apiUtils.AzureExtensionApiProvider {
     return createApiProvider([<api.AzureContainerAppsExtensionApi>{
-        // Todo: Change this to 0.0.2 later.  0.0.2 is backwards compatible anyway so this change should be fine either way.
+        // Todo: Change this to 0.0.3 later.  0.0.3 is backwards compatible anyway so this change should be fine either way.
```

Comment:
When we eventually add managed identity support for the api, I will bump the major version